script/download-standalone.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -eu
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)" # Figure out where the script is running
+. "$SCRIPT_DIR"/ci/lib/robust-bash.sh
+. "$SCRIPT_DIR"/ci/lib/utils.sh
+
+require_binary curl
+require_binary unzip
+
+STANDALONE_VERSION=$(cat ./standalone/install.ts | grep "PACT_STANDALONE_VERSION = '" | egrep -o "([0-9][\.0-9]+[0-9])")
```

Comment:
shellcheck will complain about the quotes here too, although it doesn't matter as much.

Reply:
Good reminder to re-install shell check, thanks. I removed a lot of extensions a while back because problems.